From 33efebab90a473ad62d6f48b6e5511c2b7f91434 Mon Sep 17 00:00:00 2001
From: MrHua269 <wangxyper@163.com>
Date: Tue, 11 Feb 2025 11:58:56 +0800
Subject: [PATCH] Add config for server mod name

---
 .../paper/network/PaperServerListPingEventImpl.java             | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/java/com/destroystokyo/paper/network/PaperServerListPingEventImpl.java b/src/main/java/com/destroystokyo/paper/network/PaperServerListPingEventImpl.java
index 13b03dab8..6776af992 100644
--- a/src/main/java/com/destroystokyo/paper/network/PaperServerListPingEventImpl.java
+++ b/src/main/java/com/destroystokyo/paper/network/PaperServerListPingEventImpl.java
@@ -14,7 +14,7 @@ class PaperServerListPingEventImpl extends PaperServerListPingEvent {
 
     PaperServerListPingEventImpl(MinecraftServer server, StatusClient client, int protocolVersion, @Nullable CachedServerIcon icon) {
         super(client, server.motd(), server.getPlayerCount(), server.getMaxPlayers(),
-                server.getServerModName() + ' ' + server.getServerVersion(), protocolVersion, icon);
+                me.earthme.luminol.config.modules.misc.ServerModNameConfig.fakeVanilla ? server.getServerVersion() : server.getServerModName() + ' ' + server.getServerVersion(), protocolVersion, icon);//Luminol - Fake vanilla mode
         this.server = server;
     }
 
-- 
2.55.0.windows.3

